collective/recipe/filestorage/tests/doctests.rst

```diff
@@ -16,19 +16,20 @@ Let's create and run a minimal buildout that adds an extra filestorage::
    ... user = me:pass
    ... # dead chicken .. if we don't specify no eggs, 'instance' is assumed
    ... # https://dev.plone.org/ticket/14023#comment:1
-   ... eggs = 
+   ... eggs =
    ...
    ... [filestorage]
    ... recipe = collective.recipe.filestorage
    ... parts =
    ...     my-fs
    ... ''' % globals())
-   >>> print run_buildout('-q')
+   >>> system(join('bin', 'buildout') + ' -q')
```

Comment:
Why this is not using `run_buildout` anymore?

Reply:
I check other package such collective.recipe.backup. And they don't us a similar method. Simply execute buildout directly.
Is not correct?